polkadot/runtime/westend/src/xcm_config.rs

```diff
@@ -97,6 +115,8 @@ type LocalOriginConverter = (
 	SignedAccountId32AsNative<ThisNetwork, RuntimeOrigin>,
 	// Xcm origins can be represented natively under the Xcm pallet's Xcm origin.
 	XcmPassthrough<RuntimeOrigin>,
+	// Asset Hub can gain root on the relay chain.
+	AssetHubAsSuperuser,
```

Comment:
@seadanda @karolk91 let's change this to `LocationAsSuperuser` feature: https://github.com/paritytech/polkadot-sdk/pull/8210 / https://github.com/polkadot-fellows/runtimes/pull/626/files#diff-25dd40c150a952cc9a0d97e242b9096afa7b8cac0969f07634c3a0e2fe6ba9d8R119

```suggestion
	LocationAsSuperuser<Equals<xcm_config::AssetHub>>,
```

Reply:
Nice, now that the staking mega PR is merged we can update to get those changes too.